lib/rules/index.js

```diff
@@ -74,4 +74,5 @@ module.exports = {
   'style-concatenation': require('./style-concatenation'),
   'table-groups': require('./table-groups'),
   'template-length': require('./template-length'),
+  'no-nested-splattributes': require('./no-nested-splattributes'),
```

Comment:
There's supposed to be a rule setup test to ensure this list is alphabetical but I guess that check is missing. I'll follow-up to add that check.

Reply:
hmm, interesting. I just followed what `yarn new` does :)

Reply:
Oh yeah, I'll see if I can fix that command to sort it.